persist-cli-tests/src/test/resources/test-src/input/tool_test_db_push_25/generated/rainier/generated_types.bal

```diff
@@ -0,0 +1,87 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+
+// This file is an auto-generated file by Ballerina persistence layer for rainier.
+// It should not be modified by hand.
+
+import ballerina/time;
+
+public type 'Building record {|
+    readonly 'string 'buildingCode;
+    'string 'city;
+    string 'state;
+    string 'country;
+    string 'postalCode;
+    string 'type;
+|};
+
+public type 'BuildingInsert 'Building;
+
+public type 'BuildingUpdate record {|
+    'string 'city?;
+    string 'state?;
+    string 'country?;
+    string 'postalCode?;
+    string 'type?;
+|};
+
+public type 'Department record {|
+    readonly string deptNo;
+    string deptName;
+|};
+
+public type 'DepartmentInsert 'Department;
+
+public type 'DepartmentUpdate record {|
+    string deptName?;
+|};
+
+public type 'Employee record {|
+    readonly string empNo;
+    string firstName;
+    string lastName;
+    time:Date birthDate;
+    string gender;
+    time:Date hireDate;
+    string 'departmentDeptNo;
+    string 'workspaceWorkspaceId;
```

Comment:
Do we need to escape these foreign keys?

Reply:
this escape part is coming from the escape in the field type, as we add the field type(entity name) in front to the foreign key it's automatically added to the foreign key

Reply:
updated in [ab8528a](https://github.com/ballerina-platform/persist-tools/pull/137/commits/ab8528a6d28a13e761569ec8c5a4239d4fb57963)